input/pagecontent/changes.md

```diff
@@ -27,6 +27,8 @@ This change log documents the significant updates and resolutions implemented fr
   - removed Must Support from Immunization.protocolApplied, Immunization.protocolApplied.series, Immunization.protocolApplied.targetDisease, Immunization.protocolApplied.doseNumber[x] [FHIR-44674](https://jira.hl7.org/browse/FHIR-44674), [FHIR-44656](https://jira.hl7.org/browse/FHIR-44656), [FHIR-44654](https://jira.hl7.org/browse/FHIR-44654)
   - removed Must Support from Immunization.reasonCode [FHIR-44654](https://jira.hl7.org/browse/FHIR-44654), [FHIR-45968](https://jira.hl7.org/browse/FHIR-45968)
   - added Must Support to Immunization.primarySource [FHIR-44659](https://jira.hl7.org/browse/FHIR-44659)
+  - changed Immunization.vaccineCode slicing rule from slicing by system to slice by value set [FHIR-46391](https://jira.hl7.org/browse/FHIR-46391)
+  - support for AIR codes and AMT Vaccine codes in Immunization.vaccineCode changed to define the slices in AU Core (no longer inherited from AU Base as per [FHIR-44821](https://jira.hl7.org/browse/FHIR-44821)). Slicing discriminator changed from slicing by value:system to slicing by value set and AMT value set binding is required within the AMT Vaccine slice . [FHIR-46391](https://jira.hl7.org/browse/FHIR-46391)
```

Comment:
1. Incorrect spacing here, this has ended up split over two bullet points. 
2. Last sentence has extra whitespace before the full stop.

Reply:
1. No changes made; seems to display fine now - it's on a single bullet point, split over two lines
2. Fixed 
3. Fixed